README.rst

```diff
@@ -0,0 +1,115 @@
+
+
+Monitoring-Integration
+=======================
+
+
+Functionalities
+----------------
+
+- Create a new dashboard in Grafana
+
+
+Environment Setup
+-----------------
+
+#. Clone the monitoring-integration repository
+
+    ::
+
+        $ git clone https://github.com/Tendrl/monitoring-integration.git
+	$ cd monitoring-integration
+
+
+.. note::
+
+    All the commands mentioned below are run as a regular user that has ``sudo``
+    privileges.
+    These commands are assumbed to be run from a single directory, which
+    contains the code of monitoring-integration.
+
+#. Install python pip
+
+
+   https://pip.pypa.io/en/stable/installing/
+
+
+#. Install python dependencies
+
+    ::
+
+        $ python setup.py install
+
+
+#. Install Grafana
+
+
+    http://docs.grafana.org/installation/
+
+
+#. Configure grafana for anonymous login.
+
+    ::
+
+        $ cp etc/grafana/grafana.ini /etc/grafana/.
+
+    .. note::
+
+       	  This will overwrite present grafana.ini.
+
+Prerequisites
+--------------
+
+* Grafana instance is set-up and running on the server node.
+
+* Graphite instance in set-up and running on the same server node.
```

Comment:
@rishubhjain @anivargi @anmolbabu , Now graphite is dependency to monitoring-integration, I think we need have the below config files part of this:

https://github.com/Tendrl/performance-monitoring/blob/master/etc/tendrl/performance-monitoring/graphite-web.conf.sample

https://github.com/Tendrl/performance-monitoring/blob/master/etc/tendrl/performance-monitoring/carbon.conf.sample

What you guys think?